detect_secrets/plugins/keyword.py

```diff
@@ -151,8 +151,23 @@
 # Non-greedy match
 OPTIONAL_WHITESPACE = r'\s*'
 OPTIONAL_NON_WHITESPACE = r'[^\s]{0,50}?'
-QUOTE = r'[\'"]'
-SECRET = r'[^\r\n]+'
+QUOTE = r'[\'"`]'
+'''
+Secret regex details:
+    [^\r\n]*      -> this section match with every character except line breaks.
+                     This allows to find secrets that starts with symbols or
+                     alphanumeric characters.
+    [a-zA-Z0-9]+  -> this section match only with alphanumeric characters, and at
+                     least one is required. This allows to reduce the false positives
+                     number.
+    [^\r\n]*      -> this section match with every character except line breaks.
+                     This allows to find secrets with symbols at the end.
+    [^\r\n,\'"`]  -> this section match with the last secret character that can be
+                     everything except line breaks, comma, backticks or quotes. This
+                     allows to reduce the false positives number and to prevent
+                     errors in the code snippet highlighting.
+'''
```

Comment:
Hmm, are you running `pre-commit`? I'm hitting this error:

```
$ pre-commit run --all-files
Check builtin type constructor use.......................................Passed
Check docstring is first.................................................Failed
hookid: check-docstring-first

detect_secrets/plugins/keyword.py:53 Multiple module docstrings (first docstring on line 1).

Debug Statements (Python)................................................Passed
Fix double quoted strings................................................Passed
Fix End of Files.........................................................Passed
Tests should end in _test.py.............................................Passed
Flake8...................................................................Passed
Trim Trailing Whitespace.................................................Passed
Reorder python imports...................................................Passed
Add trailing commas......................................................Passed
autopep8.................................................................Passed
Detect secrets...........................................................Failed
hookid: detect-secrets

Files were modified by this hook. Additional output:

The baseline file was updated.
Probably to keep line numbers of secrets up-to-date.
Please `git add .secrets.baseline`, thank you.


Your baseline file (.secrets.baseline) is unstaged.
`git add .secrets.baseline` to fix this.
```

Reply:
Yes, we are running `pre-commit`. This error is in the line 53 that is the comment of `FALSE_POSITIVES` variable. The details of the `SECRET` regex can be added using one line comments (`#`) instead of multiple line comments (`'''`). What do you prefer?

Reply:
Yeah, let's just do multiple line comments, so pre-commit will be happy. Then I can merge this in.

Reply:
Okay, I think that `pre-commit` is working fine now